contracts/facets/DFMoveFacet.sol

```diff
@@ -44,7 +44,7 @@ contract DFMoveFacet is WithStorage, WithArenaStorage {
         uint256 abandoning
     );
 
-    event GameStarted(uint256 timestamp);
+    event GameStarted(address startPlayer);
```

Comment:
porque

Reply:
because we can get the timestamp from the event in subgraph (it's part of the blockchain data)